src/dcp/dcp.c

```diff
@@ -271,10 +425,97 @@ int main(int argc, char** argv)
         usage = 1;
     }
 
+    char** argpaths = (&argv[optind]);
+
+#ifdef DAOS_SUPPORT
+    rc = daos_init();
+
+    /* TODO: Don't exit and fail if daos fails init,
+     * could just be regular paths */
+    if (rc != 0) {
+        MFU_LOG(MFU_LOG_ERR, "Failed to initialize daos");
+    }
+
+    struct duns_attr_t dattr  = {0}; 
+    struct duns_attr_t ddattr = {0}; 
```

Comment:
Let's move these just above daos_set_paths if they are only needed there.  Maybe even move them to be local within that function?